collectfast/strategies/base.py

```diff
@@ -97,6 +109,13 @@ def get_cache_key(self, path: str) -> str:
         path_hash = hashlib.md5(path.encode()).hexdigest()
         return settings.cache_key_prefix + path_hash
 
+    @lru_cache()
```

Comment:
```suggestion
    @lru_cache(maxsize=None)
```

Reply:
I will set it to `@lru_cache(maxsize=None)` if you really want to do this, but I am pretty sure it would work as intended with `@lru_cache(maxsize=1)` so the default of 128 provides a safety net of 127

Reply:
I guess I am quibbling over nothing.  I just checked `sys.getsizeof` on a list containing 5000 string hashes like `hashlib.md5(str(i)).hexdigest()` which is a reasonable number of static files and it is only 43kb.